HoloRepositoryDemoApplication/Assets/Scripts/ModelLoader.cs

```diff
@@ -5,67 +5,58 @@
 using UnityEngine;
 using UnityEngine.SceneManagement;
 
-namespace Microsoft.MixedReality.Toolkit.Examples.Demos.Gltf
+public class ModelLoader : MonoBehaviour
```

Comment:
It would make sense to use your Connector scripts inside the actual demo app as well. I am happy to accept eh PR before that, but I think the next step for you should be to integrate it in the demo app. Higher prio than adding tests.

Reply:
done